registration/include/pcl/registration/impl/correspondence_rejection_features.hpp

```diff
@@ -42,6 +42,7 @@
 #ifndef PCL_REGISTRATION_IMPL_CORRESPONDENCE_REJECTION_FEATURES_HPP_
 #define PCL_REGISTRATION_IMPL_CORRESPONDENCE_REJECTION_FEATURES_HPP_
 
+#include <boost/pointer_cast.hpp> // for static_pointer_cast
```

Comment:
Needs a look into to check where and why there's a boost static_pointer_cast

Reply:
That is [here](https://github.com/PointCloudLibrary/pcl/blob/master/registration/include/pcl/registration/impl/correspondence_rejection_features.hpp#L58) and five more times in that files, to convert a `shared_ptr<FeatureContainerInterface>` to a pointer to `FeatureContainer`, where [FeatureContainer inherits from FeatureContainerInterface](https://github.com/PointCloudLibrary/pcl/blob/master/registration/include/pcl/registration/correspondence_rejection_features.h#L180).